validation_testing/cdk_federation_infra_provisioning/app/package.json

```diff
@@ -13,15 +13,15 @@
   "devDependencies": {
     "@types/jest": "^29.4.0",
     "@types/node": "18.11.18",
-    "aws-cdk": "2.63.2",
+    "aws-cdk": "2.118.0",
     "jest": "^29.4.1",
     "ts-jest": "^29.0.5",
     "ts-node": "^10.9.1"
   },
   "dependencies": {
     "@aws-cdk/aws-glue-alpha": "2.63.2-alpha.0",
     "@aws-cdk/aws-redshift-alpha": "2.63.2-alpha.0",
```

Comment:
do we need this any more? given we don' use the L2 construct?

Reply:
yes we do need this. the 2.63.2 L1 construct doesn't define groupParameterName field